packages/server/src/sdk/app/rows/search/sqs.ts

```diff
@@ -376,9 +377,9 @@ export async function search(
     return response
   } catch (err: any) {
     const msg = typeof err === "string" ? err : err.message
-    if (resyncDefinitionsRequired(err.status, msg)) {
+    if (!opts?.retry && resyncDefinitionsRequired(err.status, msg)) {
       await sdk.tables.sqs.syncDefinition()
-      return search(options, table)
+      return search(options, table, { retry: true })
```

Comment:
It's that thing again where from the perspective of the caller Passing in retry true makes me think that I'm asking the function to retry But I think here you're using it as a marker to let it know that it is a retry and it shouldn't retry again I'm sure we ran into this somewhere else and we decided to use a different name, but I don't know what it was